examples/_d2n-flake-parts/flake.nix

```diff
@@ -17,10 +17,12 @@
     flake-parts.lib.mkFlake {inherit self;} {
       systems = ["x86_64-linux"];
       imports = [dream2nix.flakeModuleBeta];
-      dream2nix = {
-        config.projectRoot = ./.;
+
+      dream2nix.config.projectRoot = ./.;
```

Comment:
You could default the `projectRoot` to `self` in `flakeModule`. `self` is a module argument.